tests/sql_database/sql_source.py

```diff
@@ -165,6 +165,7 @@ def _make_precision_table(table_name: str, nullable: bool) -> Table:
                 Column("json_col", JSONB, nullable=nullable),
                 Column("bool_col", Boolean, nullable=nullable),
                 Column("uuid_col", Uuid, nullable=nullable),
+                Column("array_col", ARRAY, nullable=nullable),
```

Comment:
You need to provide `ARRAY` type in the constructor. I enabled CI for your PR all tests are failing here

Reply:
your're right, sorry ! just updated that